include/z64animation.h

```diff
@@ -12,6 +12,10 @@ struct Actor;
 struct SkelAnime;
 struct PlayerAnimationFrame;
 
+// for indexing `jointTable[]` and `morphTable[]`
+#define LIMB_INDEX_POS 0 // Translation/Offset of the entire model
+#define LIMB_INDEX_ROT 1 // Rotation of the entire model
```

Comment:
These are probably a good idea instead of having raw 0/1 every so often

Since these are about the typically-so-called "root limb" it would probably be good to put "ROOT" in there too

I'm not sure if we 100% need "INDEX" in there, if not it could be `LIMB_ROOT_POS`/ROT

I would also describe these as "position and rotation of the root limb" instead of "of the entire model" since the model will also be transformed further by (at least) the actor position and rotation into world space

Also fwiw this seems to add a lot of clutter to the PR compared to the "player" scope, I would have made it a separate PR